netboot/main.go

```diff
@@ -33,6 +37,8 @@ var (
 	readTimeout        = flag.Int("timeout", 3, "Read timeout in seconds")
 	dhcpRetries        = flag.Int("retries", 3, "Number of times a DHCP request is retried")
 	userClass          = flag.String("userclass", "", "Override DHCP User Class option")
+	caCertFile         = flag.String("cacerts", "/sys/firmware/vpd/rw/ca_certs", "Bas64 encoded CA cert file")
```

Comment:
I would use RO because otherwise someone else can modify RW with his custom CA list

Reply:
I'd initially thought I wanted certs RW, because we do update them every few months. Seems we can set RO ones with flashrom, or via the BMC anyway, so I'll move them to RO.